lib/maze/request_repeater.rb

```diff
@@ -0,0 +1,41 @@
+module Maze
+  # Repeats POST requests
+  class RequestRepeater
+
+    # @param request_type [String] The type of request being handled by this repeater
+    def initialize(request_type)
+      @request_type = request_type
+    end
+
+    # @param request [HTTPRequest] The request to be repeated
+    def repeat(request)
+
+      # TODO: Forwarding of internal errors to be considered later
+      return if request.header.keys.any? { |key| key.downcase == 'bugsnag-internal-error' }
+
+      url = URI.parse(url_for_request_type)
+      http = Net::HTTP.new(url.host)
+      bugsnag_request = Net::HTTP::Post.new(url.path)
```

Comment:
```suggestion
      http = Net::HTTP.new(@url.host)
      bugsnag_request = Net::HTTP::Post.new(@url.path)
```

Reply:
Yep, I don't know what I was up to there.